content/community.md

```diff
@@ -1,8 +1,24 @@
 ---
 title: Community
-description: Contribute to Harbor and get in touch
+description: Do you want to help build Harbor?
 ---
 
+If you’re a newcomer, check out the “[Good first issue][good-first]” and “[Help wanted][help-wanted]” labels in the Harbor repository.
+
+Read how we're using ZenHub for project and roadmap planning.
```

Comment:
I feel like this page is missing a number of links

Reply:
It is, working on that page still.

Reply:
Cool! Gotcha